app/auth.py

```diff
@@ -41,24 +42,25 @@ def registration():
     non-alphabet characters
     """
 
-    if first_name is None or first_name.strip == "" or not first_name.isalpha():
+    if first_name is None or first_name.strip() == "" or not first_name.isalpha():
```

Comment:
just use `not first_name` instead of `first_name.strip() == ""`. empty strings evaluate to `false` in python.